.github/workflows/rust.yml

```diff
@@ -59,7 +56,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [stable, beta, nightly] # No 1.29 because WASM requires Rust 1.30
+        rust: [stable, beta, nightly] # wasm-pack doesn't support rust 1.41
```

Comment:
To be pedantic but lazy, I think we should use 1.41.1 everywhere, saves trying to work out if it matters in each situation we use it.